browser/admin/src/integrator/WordBook.ts

```diff
@@ -0,0 +1,251 @@
+/* eslint-disable */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+interface DicFile {
+	headerType: string; // eg. "OOoUserDict1"
+	language: string; // the value after "lang:" (eg. "<none>")
+	dictType: string; // the value after "type:" (eg. "positive")
+	words: string[]; // list of dictionary words
+}
+
+class WordBook {
+	openDicEditor(fileName: string, dic: DicFile): void {
```

Comment:
Let's use a wordbook instead of a dic or dictionary for core standards.

Reply:
Commit : https://github.com/CollaboraOnline/online/pull/11172/commits/ec8bb9ca1ab0b729be8ad3a84bc0e96f87bb6308